### DISCUSSION TITLE
Adding Apps / Packages

### DISCUSSION THREAD

**Discussion by ma-jahn** [vercel/turborepo#1036] [Help] [answered]
Hi there,

i have create a new Turborepo (`latest`) and want to add further apps and packages.

Simply creating a directories (projects) within `apps/` or `packages/` seems not to be enough. I guess Turbo needs to be reindexed or something.

In the docs i couldn't a section for this and i thought this would be a very common scenario. Am i doing something wrong?

Thank you!

**Comment by rafaeltab** [comment, marked as answer]
Got to create a folder in apps or packages, and add a package.json in that folder. Make sure the name inside the package.json is unique

**Reply by xairoo** [reply]
> Got to create a folder in apps or packages, and add a package.json in that folder. Make sure the name inside the package.json is unique

I guess you have to update the `next.config.js` too.

```
const withTM = require("next-transpile-modules")(["ui", "somethingnew"]);
```

But that doesn't work.

A bit more detailed: https://github.com/vercel/turborepo/discussions/1107

**Reply by xairoo** [reply]
... just run `npm i` after doing this =)

**Comment by AlwaysNoobCoder** [comment]
then this should be put in the docs!
